backend/src/utils/validation/isValidQuestions.ts

```diff
@@ -0,0 +1,59 @@
+import { NOTHING } from 'config/constants';
+import { IUpdateCourseBody } from 'interfaces/ICourses/IQueryCourses';
+import { uniqWith } from 'lodash';
+
+const CORRECT_ANSWERS_AMOUNT = 1;
+
+const isValidAnswer = (answer: { aN: unknown; variant: unknown }): boolean =>
+  Boolean(
+    answer &&
+      typeof answer.aN === 'number' &&
+      answer.aN > NOTHING &&
+      answer.variant &&
+      typeof answer.variant === 'string',
+  );
+
+const isQuestionsHaveNoAnswerDuplicates = (questions: IUpdateCourseBody['test']): boolean =>
+  Boolean(
```

Comment:
i think Boolean() is odd here, what is your thought on it ?

Reply:
Here we can have `null` as the result of function. And we have to convert it to `boolean` explicitly

Reply:
ok, it's clear